oak_runtime/src/node/http/mod.rs

```diff
@@ -481,13 +480,14 @@ impl Pipe {
     }
 }
 
-// TODO(#1279): Get the label from a JSON string instead of a binary label.
 fn get_oak_label(req: &HttpRequest) -> Result<Label, OakStatus> {
     match req.headers.get(oak_abi::OAK_LABEL_HTTP_KEY) {
-        Some(label) => Label::decode(&label[..]).map_err(|err| {
-            warn!("Could not parse HTTP label: {}", err);
-            OakStatus::ErrInvalidArgs
-        }),
+        Some(label_str) => {
+            serde_json::from_str(&String::from_utf8_lossy(label_str)).map_err(|err| {
```

Comment:
Would we want to use `from_utf8` instead? It seems sensible to error if on invalid UTF-8 sequences
https://doc.rust-lang.org/std/string/struct.String.html#method.from_utf8

Reply:
I agree. I was trying to save a few error-checking lines!